docs/_guide/styles.md

```diff
@@ -162,327 +122,618 @@ class MyElement extends LitElement {
 
 </div>
 
-### Define styles in a style element {#style-element} 
+### Inheriting styles
 
-We recommend using static styles to style LitElement components. However, in some cases you may want to evaluate and apply styles per instance, rather than to all instances of a LitElement component. One way to do this is to include inline styles in a `<style>` element in your template, and use your element's properties in your CSS rules to evaluate styles per instance.
+Using an array of tagged template literals, a component can inherit the styles from a LitElement superclass, and add its own styles:
 
-<div class="alert alert-warning">
+```js
+class MyElement extends SuperElement {
+  static get styles() {
+    return [
+      super.styles,
+      css`...`
+    ];
+  }
+}
+```
 
-**Expressions inside a `<style>` element won't update per instance in ShadyCSS**. Due to limitations of the ShadyCSS polyfill, you can't use element properties in CSS rules as the expressions won't be evaluated.
+{% include project.html folder="style/superstyles" openFile="my-element.js" %}
 
-</div>
+## Shadow DOM styling overview {#shadow-dom}
+
+This section gives a brief overview of shadow DOM styling.
+
+Styles you add to a component can affect:
+
+* [The shadow tree (your component's rendered template)](#shadowroot).
+* [The component itself](#host).
+* [The component's children](#slot).
 
-```js
-import {LitElement, property} from 'lit-element';
 
+### Style the shadow tree {#shadowroot}
+
+LitElement templates are rendered into a shadow tree by default. Styles scoped to an element's shadow tree don't affect the main document. Similarly, with the exception of [inherited CSS properties](#inheritance), document-level styles don't affect the contents of a shadow tree.
+
+When you use standard CSS selectors, they only match elements in your component's shadow tree.
+
+```js
 class MyElement extends LitElement {
-  @property() mainColor = 'blue';
+  static get styles() {
+    // Write styles in standard CSS
+    return css`
+      * { color: red; }
+      p { font-family: sans-serif; }
+      .myclass { margin: 100px; }
+      #main { padding: 30px; }
+      h1 { font-size: 4em; }
+    `;
+  }
   render() {
     return html`
-      <style>
-        :host {
-          color: ${this.mainColor};
-        }
-      </style>
+      <p>Hello World</p>
+      <p class="myclass">Hello World</p>
+      <p id="main">Hello World</p>
+      <h1>Hello World</h1>
     `;
   }
 }
 ```
 
-### Define styles in an external stylesheet {#external-stylesheet}
+{% include project.html folder="style/styleatemplate" openFile="my-element.js" %}
+
+### Style the component itself
+
+Styles defined in an element's shadow tree can style the element itself (also known as the _shadow host_).
```

Comment:
is it clearer to say "style the custom element", or "style the LitElement component" rather than just "the element"?

Reply:
Hmm... Good thought. It's referring back to "Styles defined in an element's shadow tree..." so would have to rewrite the first half.

Hrm... "Styles defined in a LitElement component's shadow tree" is wordy as heck.

Styles defined in the component's shadow tree can style the component itself.

You can style the component itself with styles defined in the component's shadow tree.

I wanted to introduce the term "host" because otherwise the name of the selector is mysterious.

How about we cut to the chase and say:

You can style the component itself using special _host_ selectors. (The element that owns, or "hosts" a shadow tree is called the _host element_.)